physionet-django/console/views.py

```diff
@@ -1383,10 +1383,60 @@ def editorial_stats(request):
     approved_projects = PublishedProject.objects.all().order_by('-publish_datetime').reverse()
     rejected_projects = ArchivedProject.objects.filter(archive_reason=3).order_by('archive_datetime').reverse()
 
-    acceptance_rate = 100.0*len(approved_projects)/(len(approved_projects)+len(rejected_projects))
+    acceptance_rate = round(100.0*len(approved_projects)/(len(approved_projects)+len(rejected_projects)),2)
 
-    # Elapsed times
-    end_dict = {}
+    elapsed_times = {}
+
+    for project in approved_projects:
+
+        elapsed_times[project.title] = {}
+
+        if project.editor_assignment_datetime is not None:
+            elapsed_times[project.title]['time_sub_ed'] = \
+                abs((project.editor_assignment_datetime - \
+                project.submission_datetime).days)
+        else:
+            elapsed_times[project.title]['time_sub_ed'] = 0
+
+        if project.revision_request_datetime and project.editor_assignment_datetime is not None:
```

Comment:
Is `is not None` needed? (`None` evaluates to False)

Reply:
I suppose not, just being explicit, I can modify it

Reply:
Removing is consistent with the first clause (`if project.revision_request_datetime`) which does the same evaluation.